Common/cpp/LayoutAnimations/LayoutAnimationsManager.h

```diff
@@ -22,17 +22,13 @@ struct LayoutAnimationConfig {
   int tag;
   LayoutAnimationType type;
   std::shared_ptr<Shareable> config;
+  std::string sharedTransitionTag;
```

Comment:
Just wondering if we can make these fields `const`?

Reply:
![image](https://github.com/software-mansion/react-native-reanimated/assets/36106620/2d637814-10b7-413a-9fd9-3c052d496e27)